src/ncp/ncp_uart.cpp

```diff
@@ -287,6 +287,8 @@ void otPlatLog(otLogLevel aLogLevel, otLogRegion aLogRegion, const char *aFormat
     }
     va_end(args);
 
+    otNcpStreamWrite(0, reinterpret_cast<const uint8_t*>("\r\n"), 2);
```

Comment:
Yes, using this suffix can also solve the issue on NCP, but it will also append this additional \r\n to `syslog()` in
[`examples/platforms/posix/logging.c`](https://github.com/openthread/openthread/blob/master/examples/platforms/posix/logging.c#L72). Thoughts?